akka-sse/src/main/scala/de/heikoseeberger/akkasse/ServerSentEventParser.scala

```diff
@@ -31,31 +32,23 @@ private object ServerSentEventParser {
 
   private final val Retry = "retry"
 
-  private val linePattern = """([^:]+): ?(.*)""".r
+  private val LinePattern = """([^:]+): ?(.*)""".r
```

Comment:
So you name all your case classes lowercase then?
(Also, remember that Scala treats extractors with capitalization differently in pattern matching)